<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>301</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <column>
    <property name="text">
     <string>ID Клиента</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ФИО</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Возраст</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Выборка по:</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>99</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ФИО</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ВОЗРАСТ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID НОМЕРА</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>КЛАСС НОМЕРА</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ПЛАТА</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ДАТА ЗАЕЗДА</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ДАТА ВЫЕЗДА</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>300</y>
     <width>33</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&lt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>=</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>299</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Переключиться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>360</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать 
 clients</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>360</y>
     <width>314</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать 
 nomera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>418</y>
     <width>314</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать 
 smena</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>418</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать 
 zhurnal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>320</y>
     <width>603</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>300</y>
     <width>163</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Таблица: Клиенты</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Таблица: Сотрудники</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Таблица: Смены</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Таблица: Номера</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Таблица: Журнал</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>476</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать sotrudniki</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
